homeassistant/components/faadelays/binary_sensor.py

```diff
@@ -10,19 +11,22 @@ async def async_setup_entry(hass, entry, async_add_entities):
 
     binary_sensors = []
     for kind, attrs in FAA_BINARY_SENSORS.items():
-        name, icon = attrs
+        name = attrs[ATTR_NAME]
+        icon = attrs[ATTR_ICON]
+
         binary_sensors.append(
             FAABinarySensor(coordinator, kind, name, icon, entry.entry_id)
         )
 
     async_add_entities(binary_sensors, True)
 
 
-class FAABinarySensor(CoordinatorEntity):
+class FAABinarySensor(BinarySensorEntity, CoordinatorEntity):
```

Comment:
Order matters 😄 
```suggestion
class FAABinarySensor(CoordinatorEntity, BinarySensorEntity):
```

Reply:
Sorry for all the small fixes! Newbie here :innocent:

Reply:
Don't worry! I learn every day through reviews as well 👍